crates/forge/src/cmd/fmt.rs

```diff
@@ -157,30 +157,31 @@ impl FmtArgs {
                     let original = &source_unit.file.src;
                     let formatted = forge_fmt::format_ast(gcx, source_unit, fmt_config.clone())?;
 
-                    if original.as_str() == formatted {
-                        return None;
-                    }
-
-                    if self.check {
-                        let name =
-                            path.strip_prefix(&config.root).unwrap_or(path).display().to_string();
-                        let summary = format_diff_summary(
-                            &name,
-                            &TextDiff::from_lines(original.as_str(), &formatted),
-                        );
-                        Some(Ok(summary))
-                    } else {
-                        match fs::write(path, formatted) {
-                            Ok(_) => {
-                                let _ = sh_println!("Formatted {}", path.display());
-                                None
+                    let diff = TextDiff::from_lines(original.as_str(), &formatted);
+                    if diff.ratio() < 1.0 {
```

Comment:
is this not the same as original == formatted ?

Reply:
brain fart, pls ignore